meta-balena-common/recipes-containers/docker-disk/files/entry.sh

```diff
@@ -15,6 +15,153 @@ finish() {
 }
 trap finish EXIT
 
+source /balena-api.inc
+
+update_apps_json() {
+	local _appName
+	local _appType
+	local _api_env
+	local _apps_json
+	local _version
+	local _variant
+	local _commitID
+	local _appID
+	local _image_location
+	local _svcID
+	local _appUUID
+	local _commitID
+	local _imageId
+	local _releaseId
+	_appName="$1"
+	_appType="$2"
+	_version="$3"
+	_variant="$4"
+	_api_env="${BALENA_API_ENV}"
+	_apps_json="${DATA_VOLUME}/apps.json"
+
+	[ -z "${_appName}" ] && echo "Application name needs to be defined" && return 1
+	[ -z "${_appType}" ] && echo "Type needs to be defined" && return 1
+	[ -z "${_version}" ] && echo "Release version needs to be defined" && return 1
+	[ -z "${_variant}" ] && echo "Release variant needs to be defined" && return 1
+
+	_appID=$(print_appID_from_appName "${_appName}" "${_api_env}")
+	if [ -n "${_appID}" ] && [ "${_appID}" != "null" ]; then
+		_image_location=$(fetch_image_from_app "${_appName}" "${_version}" "${_api_env}" "${_variant}")
+		[ -z "${_image_location}" ] && return 1
+		_json=$(fetch_image_json "${_appName}" "${_api_env}" "${_version}" "${_variant}")
+		_serviceName=$(echo "${_json}" | jq --raw-output '.d[0].is_part_of__release[0].release[0].composition.services' | jq 'keys[0]' | tr -d '"')
```

Comment:
```suggestion
		_serviceName=$(echo "${_json}" | jq --raw-output '.d[0].is_part_of__release[0].release[0].composition.services | keys[0]')
```

untested, but these should be combinable via a simple pipe

Reply:
Thanks, it worked fine.